OmsAgent/omsagent_shim.sh

```diff
@@ -39,42 +28,5 @@ else
     ${PYTHON} --version
 fi
 
-# Install python-future dependency required for omsagent.py.
-# Infer distro and use the appropriate package manager,
-# falling back on direct package manager detection
-ACTION="python-future install"
-if [ -f "/etc/debian_version" ]; then # Ubuntu, Debian
-    dpkg-query -l python-future | grep ^ii
-    if [ $? != 0 ]; then
-        run_command "apt-get update" "python-future preinstall"
-        run_command "apt-get install -y python-future" $ACTION
-    fi
-elif [ -f "/etc/redhat-release" ]; then # RHEL, CentOS, Oracle
-    rpm -qi python-future
-    if [ $? != 0 ]; then
-        run_command "yum install -y python-future" $ACTION
-    fi
-elif [ -f "/etc/os-release" ]; then # Possibly SLES, openSUSE
-    grep PRETTY_NAME /etc/os-release | sed 's/PRETTY_NAME=//g' | tr -d '="' | grep -i suse
-    if [ $? != 0 ]; then
-        echo "Unsupported or indeterminable operating system" >&2
-        exit 51
-    fi
-    rpm -qi python-future
-    if [ $? != 0 ]; then
-        run_command "zypper --non-interactive install python-future" $ACTION
-    fi
-elif [ -x "$(command -v apt-get)" ]; then
-    run_command "apt-get update" "python-future preinstall"
-    run_command "apt-get install -y python-future" $ACTION
-elif [ -x "$(command -v yum)" ]; then
-    run_command "yum install -y python-future" $ACTION
-elif [ -x "$(command -v zypper)" ]; then
-    run_command "zypper --non-interactive install python-future" $ACTION
-else
-    echo "Unsupported or indeterminable operating system" >&2
-    exit 51
-fi
-
-${PYTHON} ${COMMAND} ${ARG}
+PYTHONPATH=${PWD}/ext/future:${PYTHONPATH} ${PYTHON} ${COMMAND} ${ARG}
```

Comment:
Along with `PYTHONPATH`, do you need to also set up `PATH`?

Reply:
Doesn't seem necessary from my testing